roles/base/tasks/main.yml

```diff
@@ -43,13 +43,13 @@
     dest: /usr/local/
 - name: install Golang v1.5.1
   shell: chdir=/usr/local/ creates=/usr/local/go/bin/go tar xfvz go1.5.1.linux-amd64.tar.gz
-- name: download etcd v2.1.1
+- name: download etcd v2.2.2
   get_url: 
     validate_certs: no
-    url: https://github.com/coreos/etcd/releases/download/v2.1.1/etcd-v2.1.1-linux-amd64.tar.gz
+    url: https://github.com/coreos/etcd/releases/download/v2.2.2/etcd-v2.2.2-linux-amd64.tar.gz
```

Comment:
@erikh do we know if this version bump is earning us something. My concern is that it might have impact on volplugin and netplugin as they largely depend on it and we have been doing fine so far with v2.1. Is it something we need feedback from @shaleman or @jainvipin? Otherwise, I am ok with this version bump.